api/v1beta1/webhook_suite_test.go

```diff
@@ -66,7 +64,15 @@ var _ = BeforeSuite(func() {
 		},
 	}
 
-	cfg, err := testEnv.Start()
+	var cfg *rest.Config
```

Comment:
Hmm... V2 requires an explicit GinkgoRecover here where it wasn't required prior?

Reply:
Correct, see the discussion in https://github.com/onsi/ginkgo/issues/882. Basically it sounds like this is how we can enforce a timeout on an async test